bigbone/src/main/kotlin/social/bigbone/api/entity/Preferences.kt

```diff
@@ -0,0 +1,51 @@
+package social.bigbone.api.entity
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import social.bigbone.api.entity.data.Visibility
+
+/**
+ * Represents a Preferences about an account.
+ * @see <a href="https://docs.joinmastodon.org/entities/Preferences/">Mastodon API Preferences</a>
+ */
+@Serializable
+data class Preferences(
+    /**
+     * Default visibility for new posts.
+     */
+    @SerialName("posting:default:visibility")
+    val defaultVisibility: String = Visibility.PUBLIC.value,
```

Comment:
I think we should use type `Visibility` here instead of `String`.

Reply:
Hence, I think I should modifiy also visibility attribute within Status.kt, right?

Reply:
Yes, that would make sense. Thx!

Reply:
I refactored Visibility dependency use anywhere in the code to avoid ".value"